src/Scan/BasePolling.ts

```diff
@@ -33,9 +33,7 @@ export class BasePolling implements Polling {
     }
 
     if (this.options.interval) {
-      const interval = this.toMilliseconds(this.options.interval);
```

Comment:
Is there any reason to keep this unused method?

Reply:
yes, the reason is we already pass milliseconds, no need to use `toMilliseconds` here

Reply:
Yet you claim opposite — if you don't need it, why is it still there? Perhaps it's used elsewhere, but why doesn't the same statment apply to them? For example, `timeout` you pass is now a number, not a string.

You should either remove this method entirely and adjust `PollingConfig` accordingly, or avoid converting the string with your custom converter and instead rely on the more robust and tested `ms` package, doing this in the concrete command rather than globally.

Reply:
done!

Reply:
I see, removed the `toMilliseconds` and changed the type of `timeout` to number